index.html

```diff
@@ -36,7 +35,6 @@
         <div class="content">
 
          
-            <!-- a video starts -->
               <div class="video">
                  <div class="thumbnail">
                   <iframe width="685" height="415" src="https://www.youtube.com/embed/lJIrF4YjHfQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
- [ ] You did a great job @maths29  😃, however, kindly make sure to add `why` the student should make a change in all comments you gave 👍 
- [ ] Also, it could be better to add emojis 😄 ,  to make your comment more friendly, Kindly add it 👍